web/src/app/webview/src/debug-main.ts

```diff
@@ -8,8 +8,8 @@ import { SourcemappedWorker } from '@keymanapp/lexical-model-layer/web'
   * This can only be done during load when the active script will be the
   * last script loaded.  Otherwise the script must be identified by name.
  */
- var scripts = document.getElementsByTagName('script');
- var ss = scripts[scripts.length-1].src;
- var sPath = ss.substr(0,ss.lastIndexOf('/')+1);
+const scripts = document.getElementsByTagName('script');
+const ss = scripts[scripts.length-1].src;
+const sPath = ss.slice(0,ss.lastIndexOf('/'));
```

Comment:
I think this needs to be:
```suggestion
const sPath = ss.slice(0,ss.lastIndexOf('/')+1);
```

That is:
```
let path='webview/src/debug-main.ts';
path.substr(0, path.lastIndexOf('/')+1) == 'webview/src/'
path.slice(0, path.lastIndexOf('/')+1) == 'webview/src/'
```

Reply:
That would be wrong. The second parameter of `slice` is the index of the last element to include, not the number of elements.

Reply:
I tried it tho